mappings/net/minecraft/world/GameRules.mapping

```diff
@@ -20,97 +41,115 @@ CLASS net/minecraft/unmapped/C_xmldumst net/minecraft/world/GameRules
 	FIELD f_tputawyr DO_MOB_SPAWNING Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;
-		COMMENT A {@linkplain Rule game rule} which regulates whether mobs can spawn naturally.
+		COMMENT A {@linkplain AbstractGameRule game rule} which regulates whether mobs can spawn naturally.
+	FIELD f_ulkaqxnx DO_DAYLIGHT_CYCLE Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;
+	FIELD f_uqgkyhqg FALL_DAMAGE Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;
+	FIELD f_uwzhnovj PLAYERS_SLEEPING_PERCENTAGE Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;
 	FIELD f_vimssrzh DO_MOB_LOOT Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;
-		COMMENT A {@linkplain Rule game rule} which regulates whether mobs should drop loot on death.
+		COMMENT A {@linkplain AbstractGameRule game rule} which regulates whether mobs should drop loot on death.
+	FIELD f_vvenpozn MAX_COMMAND_CHAIN_LENGTH Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;
+	FIELD f_wvejngfp NATURAL_REGENERATION Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;
 	FIELD f_xfpasujp REDUCED_DEBUG_INFO Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;
-		COMMENT A {@linkplain Rule game rule} which regulates whether clients' {@linkplain net.minecraft.client.gui.hud.DebugHud debug HUD}s show reduced information.
+		COMMENT A {@linkplain AbstractGameRule game rule} which regulates whether clients' {@linkplain net.minecraft.client.gui.hud.DebugHud debug HUD}s show reduced information.
 		COMMENT
 		COMMENT <p>When the value of this rule is changed, all connected clients will be notified to update their display.
 		COMMENT In vanilla, this includes the visibility of coordinates on the clients' debug HUDs.
 	FIELD f_xhcxhtfb ANNOUNCE_ADVANCEMENTS Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;
-		COMMENT A {@linkplain Rule game rule} which regulates whether a player's advancements should be announced in chat.
+		COMMENT A {@linkplain AbstractGameRule game rule} which regulates whether a player's advancements should be announced in chat.
 	FIELD f_zumpbxtd MAX_ENTITY_CRAMMING Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;
-		COMMENT A {@linkplain Rule game rule} which regulates the number of entities that can be crammed into a block space before they incur cramming damage.
+		COMMENT A {@linkplain AbstractGameRule game rule} which regulates the number of entities that can be crammed into a block space before they incur cramming damage.
+	FIELD f_zwlbxxnu GLOBAL_SOUND_EVENTS Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;
 	METHOD <init> (Lcom/mojang/serialization/DynamicLike;)V
-		ARG 1 rules
+		ARG 1 gameRules
 	METHOD <init> (Ljava/util/Map;)V
-		ARG 1 rules
+		ARG 1 gameRules
 	METHOD m_annnbzfc setAllValues (Lnet/minecraft/unmapped/C_xmldumst;Lnet/minecraft/server/MinecraftServer;)V
-		ARG 1 rules
+		ARG 1 other
 		ARG 2 server
-	METHOD m_dpsrpyxn getInt (Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;)I
-		ARG 1 rule
+	METHOD m_dpsrpyxn getIntGameRuleValue (Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;)I
```

Comment:
```suggestion
	METHOD m_dpsrpyxn getInt (Lnet/minecraft/unmapped/C_xmldumst$C_iuaedxxw;)I
```
not loving these changes, they're just *so* verbose

Reply:
`getIntValue`?

Reply:
given their parameters, I think just `getValue` is fine for both methods

Reply:
I would have thought I'd have named it that if I could, I'll see if there was a reason why not.  If not, then I will do that.  Much more sensible than the name I currently have.

Reply:
Yeah, I can't do that because they both take a `GameRules.Key` as their only argument.  At least, Enigma doesn't let me.

Reply:
Ah, since that's not possible, I think `getIntValue` is good.